src/EnergyPlus/SetPointManager.cc

```diff
@@ -3917,6 +3921,16 @@ namespace SetPointManager {
 						// Call the CALC routine, with an optional argument to only set
 						// the initialization NODE(:)% setpoint, and not the OutAirSetPtMgr(:)%SetPt
 						OutAirSetPtMgr( SetPtMgrNum ).calculate( NodeNum, true );
+						Node( NodeNum ).TempSetPointLo = OutAirSetPtMgr( SetPtMgrNum ).SetPtLo;
```

Comment:
@Nigusse There's a misundertanding of the intent here.  The current OA setpoint manager calculates a setpoint value somewhere between SetPtLo and SetPtHi (inclusive).  That's the value that should be used, no matter which control variable is selected.  For iCtrlVarType_Temp it gets place on Node.TempSetpoint.  For iCtrlVarType_MaxTemp it gets placed on Node.TempSetPointHi, and for iCtrlVarType_MinTemp it gets placede on TempSetPointLo.  Looking at the calculate function, you should probably drop the second argument (true here) and put the correct assignments in this section.